pelicanconf.py

```diff
@@ -68,6 +68,8 @@
 YOUTUBEURL = 'http://youtube.com/osuopensourcelab'
 GPLUSURL = 'https://plus.google.com/107361178205293595706?rel=author'
 
+GOOGLE_ANALYTICS = 'UA-537692-18'
```

Comment:
Oh! We have slightly different codes per subdomain. TIL. I just grabbed the first one alphabetically, but I suppose this makes sense.